oeps/architectural-decisions/oep-0064-mobile-application-codebase-modernization.rst

```diff
@@ -0,0 +1,103 @@
+OEP-64: Mobile App Codebase Modernization
+#############################################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-64 <oep-0064-mobile-application-codebase-modernization>`
+   * - Title
+     - Mobile Application Codebase Modernization
+   * - Last Modified
+     - 2023-06-09
+   * - Authors
+     - 
+       * Marco Morales <marco@schema.education>
+       * Volodymyr Chekyrta <volodymyr.chekyrta@raccoongang.com>
+   * - Arbiter
+     - Ed Zarecor <ezarecor@axim.org>
+   * - Status
+     - Under Review
+   * - Type
+     - Architecture Decision
+   * - Created
+     - 2023-06-09
+   * - Review Period
+     - 2023-06-15 - 2023-08-01
+   * - References
+     - `Related Funded Contribution <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3663429666/FC-0011+-+Mobile+Product+Strategy+Backlog+Development>`_
+
+.. contents::
+  :local:
+  :depth: 1
+
+Abstract
+********
+
+As part of the discovery effort done for a funded contribution (FC-0011) we are recommending that the Open edX platform adopt newly built iOS and Android applications as the new default mobile applications for the platform moving forward, starting with the next named release. These new applications are rebuilt with MVVM architectural patterns and are fully rebuilt in Swift and Kotlin respectively. These new code bases should facilitate faster development cycles, improved pluggability and community contribution rates. Currently not all functionality is 100% identical to the existing mobile applications, but efforts are underway to improve pluggability and close common community requirements for the mobile experience.
+
+Motivation
+**********
+
+The motivation for this shift comes from an opportunistic review of newly rebuilt mobile applications from Raccoon Gang that seek to remove some of the mobile contribution impediments that the existing applications have. The original mobile application repositories were authored in 2015 and have progressively started including new architectural patterns and the use of Swift / Kotlin, but challenges remain for contribution and coded clarity. Additionally, coding patterns, libraries, and architectural patterns have changed significantly since the original authorship of our mobile applications, and the newly rebuilt Raccoon Gang applications are open sourced and a great starting point for renewed emphasis on mobile learning options for Open edX platform adopters and learners.
+
+Additionally, Axim Collaborative is looking to expand the way the Open edX mobile applications can serve as a tool for expanded access to learning opportunities. Mobile application learning is a key requirement to modern learning platforms and we believe that with concentrated effort new contributions and improvements in the mobile space will help expand the use of the open edx platform.
```

Comment:
```suggestion
Additionally, Axim Collaborative is looking to expand the way the Open edX mobile applications can serve as a tool for expanded access to learning opportunities. Mobile application learning is a key requirement to modern learning platforms and we believe that with concentrated effort new contributions and improvements in the mobile space will help expand the use of the Open edX platform.
```

Why call out Axim Collaborative? Is this not a community goal?

Reply:
Yes, I had meant to comment to that effect as well.  Let's focus on the general reasons:

Something like: 

A robust mobile experience makes learning opportunities more accessible whether that is for learners who don't have ready access to personal computers, or need to fit learning into their schedules around work and family commitments.